grid2op/Space/GridObjects.py

```diff
@@ -513,6 +514,7 @@ class GridObjects:
 
     sub_info : ClassVar[np.ndarray] = None
     dim_topo : ClassVar[np.ndarray] = -1
+    allow_shedding : ClassVar[bool] = DEFAULT_ALLOW_SHEDDING
```

Comment:
If you add class argument here you probably need to implement:
- `make_cls_dict` (not sure you did)
- `from_dict` (which I believe you did but i'm not sure)
- `_get_full_cls_str` (almost certain you did not do it)

Yeah I know adding stuff in grid2op can be a pain :-( lots of glue code everywhere :-(

Reply:
My ethos was to change as few things as possible, since it will break most unit tests as well (for instance if I include allow_sheedding in the dict representation it will break a bunch of unit tests that are checking against a stored Dict that does not include that attribute yet). Is there some way we could make those unit tests more dynamic (i.e. not hard-code the dictionary, string, etc. to compare to)?

Reply:
I specifically want these unit tests to be broken when some variables are added to the GridObjects. So that I can remember to change a few things here and there.

Yes there would be ways, but I prefer not to, because adding these kind of attributes should be done with extreme caution :-)
(you probably need to change something in the test_Action.py and test_Observation.py and that's it if I remember correctly)

Reply:
I see if it is intentional for those to break then it would seem they are successfully serving their function, will try to address this soon then!